### PR TITLE
Add new "Quickstart" section to README.md

### PR DESCRIPTION
**Pull request by prakashsurya**

### CODE REVIEW

**Comment by prakashsurya** [comment]
See [here](https://github.com/prakashsurya/appliance-build/blob/quickstart/README.md#quickstart-for-the-impatient) for an example of what this looks like.

File: README.md [hunk 1]
```diff
@@ -9,6 +9,25 @@ machine images containing the Delphix Dynamic Data Platform, that are
 capable of running in cloud and non-cloud hypervisors alike (e.g. Amazon
 EC2, Microsoft Azure, VMware, OpenStack).
 
+## Quickstart (for the impatient)
+
+Run this commands on "dlpxdc.co" to create the VM used to do the build:
```
Comment (ahrens) [inline README.md]: should be `this command`

Comment (prakashsurya) [reply README.md]: done; thanks!

**Pull request merged**